core/dplug/core/ringbuf.d

```diff
@@ -12,8 +12,10 @@ import dplug.core.sync;
 import dplug.core.nogc;
 import dplug.core.vec;
 
+deprecated("Use makeRingBufferNoGC instead.") // unfortunately no warning because of https://issues.dlang.org/show_bug.cgi?id=19780
+alias ringBufferNoGC = makeRingBufferNoGC;
```

Comment:
> If you change a name, please add a

I'm doing so :see_no_evil:

Reply:
Oops, sorry, I said it while I had read only one of the commits :|

Reply:
no problem, thx for the clarification :)

Reply:
Let me know when you want me to pull :)